wiki/Tournaments/RMoT_Invitational/1/en.md

```diff
@@ -0,0 +1,115 @@
+# Rapid Monthly osu! Tournament 1
+
+The Rapid Monthly osu! Tournament Invitational \#1 is a tournament organized and ran by the Rapid Tournaments team.
+It is an exclusive tournament where only players who reached semi-finals in one of the last 4 RMoT's get invited.
+
+The tournament brackets can be found [on Challonge.](https://challonge.com/rmotinvitational1). Most of the information here complies with the [forum post](https://osu.ppy.sh/forum/t/538852).
+
+
+![][flag_US] [\[Sony\]](https://osu.ppy.sh/u/8750694) (formerly known as sonyguy699) won the Invitational, beating ![][flag_PL] [CarlosH20](https://osu.ppy.sh/u/8102219) in the finals. 
+
+## Schedule
+
+The RMoT Invitational is held over two weekends.
+
+| Event                               | Date        |
+|------------------------------------:|:------------|
+| Round of 16                         | 7. Jan 2017  |
+| Quarterfinals and Losers Round 1    | 8. Jan 2017  |
+| Semifinals and Losers Round 2 and 3 | 14. May 2017 |
+| Losers Round 4, 5 and 6             | 15. Jan 2017 |
+| Finals and Grand Finals             | 15. Jan 2017 |
+
+## Staff
+
+| Role              | Member |
+|-------------------|--------|
+| Host              | ![][flag_SI] [Redavor](https://osu.ppy.sh/u/3328606)|
+| Admins            | ![][flag_SI] [DomenBzo](https://osu.ppy.sh/u/3450825), ![][flag_SI] [kristi71111](https://osu.ppy.sh/u/3826105), ![][flag_TR] [oralekin](https://osu.ppy.sh/u/7631823)|
+| Referees          | ![][flag_US] [Rosilius](https://osu.ppy.sh/u/5939931), ![][flag_US] [Obilitus](https://osu.ppy.sh/u/4857153), ![][flag_US] [Cierius](https://osu.ppy.sh/u/7376116)|
+| Mappool Selectors | ![][flag_US] [Rosilius](https://osu.ppy.sh/u/5939931), ![][flag_US] [Obilitus](https://osu.ppy.sh/u/4857153), ![][flag_US] [Cierius](https://osu.ppy.sh/u/7376116)|
+| Streamers         | ![][flag_US] [Rosilius](https://osu.ppy.sh/u/5939931), ![][flag_SI] [DomenBzo](https://osu.ppy.sh/u/3450825)|
+
+## Prizes
+
+The Prizes are handed out in the form of Points. every 36 points can be redeemed for a month of osu!supporter. Invitational winners are awarded twice the amount awarded in RMoT's.
+The winner of the Invitational tournament gets 96 points, 2ⁿᵈ place gets 48 points etc..
```

Comment:
Sup.

Also, how do you calculate the points. Divide by two?

`The winner of the Invitational tournament gets 96 points, 2nd place gets 48 points, 3rd place gets 24 points, and the points continuously divided by two the lower the placement until the points is 0.`

Not comfortable with my last line. Might as well list out the points for placement.